### ISSUE TITLE
20 HTML attributes missing in json

### ISSUE THREAD

**Issue by titoBouzout** [mdn/browser-compat-data#25075]
### What type of issue is this?

Other

### What information was incorrect, unhelpful, or incomplete?

I have been working on this table https://potahtml.github.io/namespace-jsx-project/index.html It's a mash-up of frameworks namespace JSX, browser setters (chrome, firefox), vscode html LSP, and MDN

The data from MDN is missing some information:

1. Missing deprecations

For example: When reading `èlement.json` such for `style` to be https://github.com/mdn/browser-compat-data/blob/main/html/elements/style.json then attribute `scoped` is missing in the JSON.  I can somewhat confirm its deprecated because theres a note here (CTRL+F for "scoped") https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes

The list of attributes I could confirm that are deprecated but missing in `$element.json` are the following:

```text
// deprecated https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes
'img.HTMLImageElement.intrinsicsize': true,

// deprecated https://developer.mozilla.org/en-US/docs/Web/HTML/Element/col#bgcolor
'col.HTMLTableColElement.bgcolor': true,

// deprecated https://developer.mozilla.org/en-US/docs/Web/HTML/Element/colgroup#bgcolor
'colgroup.HTMLTableColElement.bgcolor': true,

// deprecated https://developer.mozilla.org/en-US/docs/Web/HTML/Element/script#language
'script.HTMLScriptElement.language': true,

// deprecated https://developer.mozilla.org/en-US/docs/Web/HTML/Element/form#accept
'form.HTMLFormElement.accept': true,

// SVG

// deprecated https://developer.mozilla.org/en-US/docs/Web/API/SVGStyleElement
'style.SVGStyleElement.type': true,

// MATHML

// deprecated https://developer.mozilla.org/en-US/docs/Web/MathML/Element/annotation-xml
'annotation-xml.MathMLElement.src': true,

// deprecated https://developer.mozilla.org/en-US/docs/Web/MathML/Element/annotation
'annotation.MathMLElement.src': true,

// deprecated https://developer.mozilla.org/en-US/docs/Web/MathML/Element/ms
'ms.MathMLElement.lquote': true,
'ms.MathMLElement.rquote': true,

// legacy deprecated https://developer.mozilla.org/en-US/docs/Web/MathML/Element/mstyle
'mstyle.MathMLElement.scriptminsize': true,
'mstyle.MathMLElement.scriptsizemultiplier': true,

// deprecated https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes
'style.HTMLStyleElement.scoped': true,
'style.SVGStyleElement.scoped': true,

```

2. Missing information

I couldn't find information about the following attributes, but I am pretty sure these are deprecated/obsolete.

```text
// not found in mdn
'iframe.HTMLIFrameElement.allowtransparency': true,
```

about the following, I am not that sure any more, but I suppose I have found these are deprecated

```
'iframe.HTMLIFrameElement.seamless': true,

'object.HTMLObjectElement.typemustmatch': true,

'menu.HTMLMenuElement.label': true,
'menu.HTMLMenuElement.type': true,
````

3. Missing Tags

I couldnt find a `$element.json` file for the following tags. There are other deprecated tags such `frameset.json`. So I suppose it would be only consistent to add them, maybe.

```
'applet',
'basefont',
'bgsound',
'blink',
'listing',
'isindex',
'keygen',
'menuitem',
'multicol',
'nextid',
'noindex',
'spacer',

'content',
'portal',
'image',
'imagee',
'shadow',
```

4. The following have the experimental bit set, but I am not convinced these are still  experimental 

```
a.SVGAElement.ping  
script.SVGScriptElement.async 
script.SVGScriptElement.defer 
```

### What did you expect to see?

-

### Did you test this? If so, how?

-

**Comment by caugner** [comment]
tl;dr I checked all mentioned features, and as far as I can see, everything seems aligned with BCD's [data guidelines](https://github.com/mdn/browser-compat-data/tree/main/docs/data-guidelines).

Please note BCD's conditions for [removal of irrelevant features](https://github.com/mdn/browser-compat-data/blob/main/docs/data-guidelines/index.md#removal-of-irrelevant-features).

## 1. Missing deprecations

These were mostly removed from BCD as irrelevant.

> 'img.HTMLImageElement.intrinsicsize': true,

🗑  Removed in https://github.com/mdn/browser-compat-data/pull/15997.

> 'col.HTMLTableColElement.bgcolor': true,
> 'colgroup.HTMLTableColElement.bgcolor': true,

🗑 Removed in https://github.com/mdn/browser-compat-data/pull/21958.

> 'script.HTMLScriptElement.language': true,

🗑 Removed in https://github.com/mdn/browser-compat-data/pull/21916.

> 'form.HTMLFormElement.accept': true,

🗑 Removed in https://github.com/mdn/browser-compat-data/pull/21831.

### SVG

> 'style.SVGStyleElement.type': true,

✅ Added in https://github.com/mdn/browser-compat-data/pull/8255, and [still exists](https://github.com/mdn/browser-compat-data/blob/v6.0.13/api/SVGStyleElement.json#L240-L287).

### MATHML

> 'annotation-xml.MathMLElement.src': true,
> 'annotation.MathMLElement.src': true,

✅ Added in https://github.com/mdn/browser-compat-data/pull/23426.

> 'ms.MathMLElement.lquote': true,
>'ms.MathMLElement.rquote': true,

🗑 Removed in https://github.com/mdn/browser-compat-data/pull/25108.

> 'mstyle.MathMLElement.scriptminsize': true,
> 'mstyle.MathMLElement.scriptsizemultiplier': true,

🗑 Removed in https://github.com/mdn/browser-compat-data/pull/23962.

> 'style.HTMLStyleElement.scoped': true,
> 'style.SVGStyleElement.scoped': true,

🗑 Removed in https://github.com/mdn/browser-compat-data/pull/18258.

## 2. Missing information

> 'iframe.HTMLIFrameElement.allowtransparency': true,

❔ Never mentioned in mdn/content, and not in mdn/browser-compat-data, but I found [this mention in a Chrome 83 commit](https://github.com/chromium/chromium/commit/e5c2176009f5e82d1dc702348d3f1cdfe51fc4dd#diff-ffb4e1149e06afdc6a9e0ac855091f3d15a935a3483ae6a0ebbf17b8764ccf08R17).

> 'iframe.HTMLIFrameElement.seamless': true,

❔ Never mentioned in mdn/content, and not in mdn/browser-compat-data.

> 'object.HTMLObjectElement.typemustmatch': true,

🗑 Removed in https://github.com/mdn/browser-compat-data/pull/9671.

> 'menu.HTMLMenuElement.label': true,
> 'menu.HTMLMenuElement.type': true,

Removed in https://github.com/mdn/browser-compat-data/pull/18981.

## 3. Missing Tags

> 'applet',

🗑 Removed in https://github.com/mdn/browser-compat-data/pull/19645.

> 'basefont',

🗑 Removed in https://github.com/mdn/browser-compat-data/pull/16973.

> 'bgsound',
> 'blink',

🗑 Removed in https://github.com/mdn/browser-compat-data/pull/19655.

> 'listing',
> 'isindex',

🗑 Removed in https://github.com/mdn/browser-compat-data/pull/10551.

> 'keygen',

🗑 Removed in https://github.com/mdn/browser-compat-data/pull/19655.

> 'menuitem',

🗑 Removed in https://github.com/mdn/browser-compat-data/pull/18983.

> 'multicol',

🗑 Removed in https://github.com/mdn/browser-compat-data/pull/10545.

> 'nextid',

🗑 Removed in https://github.com/mdn/browser-compat-data/pull/10550.

> 'noindex',

❔ Never mentioned in mdn/content, and not in mdn/browser-compat-data. Maybe a misunderstanding about the `noindex` value for `robots`?

> 'spacer',

🗑 Removed in https://github.com/mdn/browser-compat-data/pull/19739.

> 'content',

🗑 Removed in https://github.com/mdn/browser-compat-data/pull/19147.

> 'portal',

🗑 Removed in https://github.com/mdn/browser-compat-data/pull/25805.

> 'image',

🗑 Removed in https://github.com/mdn/browser-compat-data/pull/21915.

> 'imagee',

❔ Never mentioned in mdn/content, and not in mdn/browser-compat-data. Maybe just a typo?

> 'shadow',

🗑 Removed in https://github.com/mdn/browser-compat-data/pull/19293.

## 4. The following have the experimental bit set, but I am not convinced these are still experimental

> a.SVGAElement.ping

✅ Not marked as experimental since the feature was added in https://github.com/mdn/browser-compat-data/pull/8107/files#diff-d4754ef580d5eb3b76f6ca3c6a8590c4fb9d7bc239972e46da846f5085a39f00R331.

> script.SVGScriptElement.async 
> script.SVGScriptElement.defer 

✅ According to our data (see [the BCD table](https://developer.mozilla.org/en-US/docs/Web/API/SVGScriptElement#browser_compatibility)), these are only supported in Firefox, and therefore experimental as [per our guidelines](https://github.com/mdn/browser-compat-data/blob/main/docs/data-guidelines/index.md#setting-experimental).

**Comment by titoBouzout** [comment]
Thanks for double-checking @caugner. Its very unfortunate deprecations are not kept in some way, because when you find some of these properties, it becomes some sort of hunt trying to see what's going on with it.
